### ISSUE TITLE
[v2.0.0] TypeError: merge.smart is not a function

### ISSUE THREAD

**Issue by antoinerousseau** [netlify/netlify-lambda#249]
I just upgraded from 1.6.3 to 2.0.0 and now the `--config` arg crashes with:

```
yarn run v1.22.4
$ cross-env NODE_ENV=development netlify-lambda serve src/functions --config webpack.functions.js --timeout 30
netlify-lambda: Starting server
/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/lib/build.js:148
    return merge.smart(webpackConfig, webpackAdditional);
                 ^

TypeError: merge.smart is not a function
    at webpackConfig (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/lib/build.js:148:18)
    at Object.exports.watch (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/lib/build.js:170:26)
    at Command.<anonymous> (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/bin/cmd.js:61:11)
    at Command.listener [as _actionHandler] (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/node_modules/commander/index.js:413:31)
    at Command._parseCommand (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/node_modules/commander/index.js:914:14)
    at Command._dispatchSubcommand (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/node_modules/commander/index.js:865:18)
    at Command._parseCommand (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/node_modules/commander/index.js:882:12)
    at Command.parse (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/node_modules/commander/index.js:717:10)
    at Object.<anonymous> (/Users/antoine/Sites/oss/gatsby-starter-antoine/node_modules/netlify-lambda/bin/cmd.js:115:9)
    at Module._compile (internal/modules/cjs/loader.js:1147:30)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by briangonzalezmia** [comment]
I can confirm this is an issue with 2.0.0. Reverting back to 1.6.3 works.

**Comment by bloqhead** [comment]
Chiming in. Switching back to the previous version of netlify-lambda worked for me, but this is obviously not ideal.

**Comment by svengau** [comment]
merge.smart(..) has been removed from webpack-merge@5.0.3:

https://github.com/survivejs/webpack-merge/blob/master/CHANGELOG.md

**Comment by svengau** [comment]
@erezrokah you've updated packages in this commit: https://github.com/netlify/netlify-lambda/commit/f55897692615b931b2de4f02a0491cf52474a2a3

Maybe we can downgrade to previous version of webpack-merge?

**Comment by erezrokah** [comment]
A fix for this is now released in `v2.0.1`.
Sorry for the late follow up, and thanks everyone for reporting the issue.
